<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlutoPropAssessor</class>
 <widget class="QMainWindow" name="PlutoPropAssessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>139</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>342</width>
    <height>139</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>342</width>
    <height>139</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PLUTO Proprioception Assessment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pbSubject">
       <property name="toolTip">
        <string>Select the subject to assess</string>
       </property>
       <property name="text">
        <string>Select Subject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCalibration">
       <property name="text">
        <string>Calibrate PLUTO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbTestDevice">
       <property name="text">
        <string>Test Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPropAssessment">
       <property name="text">
        <string>Proprioception Assessment</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
